help/availability.md

```diff
@@ -5,7 +5,7 @@ Thursday, with no announcements Friday or Saturday. Check the [current
 time](https://arxiv.org/localtime) at the arXiv main site, with the
 amount of time remaining before the deadline. Submissions are made public as part of the scheduled announcement process. This includes new submissions as well as [replacements](/help/replace), [withdrawal notices](/help/withdraw), [cross listings](/help/cross) and [journal reference](/help/jref). Journal reference submissions do not appear in the mailings, but are still processed at that time. 
 
-<a href="#id-before"></a>
+<a href="id-before"></a>
```

Comment:
maybe should be
<span id="id-before"></span>
or
< name="id-before"></a>

Reply:
it should either be name or converted to a span. I verified that using it as a name works in a docker container, so just doing that for now.